### PR TITLE
Applied latest gdnative structural changes to our simple demo

### PR DESCRIPTION
**Pull request by BastiaanOlij**
And now it works with Godot 3 beta (only tested on osx)

### CODE REVIEW

**Comment by karroffel** [comment]
Looking good, will test on Linux in a second!

File: c/SimpleDemo/main.gd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 extends Control
 
 # load the SIMPLE library
-onready var data = preload("res://SIMPLE.gdns").new()
```
Comment (karroffel) [inline c/SimpleDemo/main.gd]: it's `res://bin/simple.gdns` now.

**Comment by karroffel** [comment]
Works on Linux and Windows (MinGW)

File: c/SimpleDemo/main.gd (hunk 1 above)
Comment (BastiaanOlij) [reply c/SimpleDemo/main.gd]: oops, yeah I didn't like the upper case

**Pull request merged**